nms/package.json

```diff
@@ -157,10 +166,12 @@
     "refractor/prismjs": "^1.27.0",
     "jspdf": "^2.3.1",
     "cacache/ssri": "6.0.2",
-    "@types/react": "^16.8.6"
+    "@types/react": "^16.8.6",
+    "@types/webpack": "^4.41.32",
+    "webpack": "^4.46.0"
   },
   "scripts": {
-    "eslint": "./node_modules/.bin/eslint --ignore-path .eslintignore",
+    "eslint": "./node_modules/.bin/eslint --fix --ignore-path .eslintignore",
```

Comment:
Do we really want that?
It might be better to have a separate "script" `eslint-fix` to indeed change code.